pkg/sql/logictest/testdata/logic_test/aggregate

```diff
@@ -1349,86 +1351,125 @@ CREATE TABLE statistics_agg_test (
   y float,
   x float,
   int_y int,
-  int_x int
+  int_x int,
+  dy decimal,
+  dx decimal
 )
 
 statement OK
-INSERT INTO statistics_agg_test (y, x, int_y, int_x) VALUES
-  (1.0,   10.0,    1,   10),
-  (2.0,   25.0,    2,   25),
-  (2.0,   25.0,    2,   25),
-  (3.0,   40.0,    3,   40),
-  (3.0,   40.0,    3,   40),
-  (3.0,   40.0,    3,   40),
-  (4.0,  100.0,    4,  100),
-  (4.0,  100.0,    4,  100),
-  (4.0,  100.0,    4,  100),
-  (4.0,  100.0,    4,  100),
-  (NULL,  NULL, NULL, NULL)
-
-query FFFF
-SELECT corr(y, x)::decimal, corr(int_y, int_x)::decimal, corr(y, int_x)::decimal, corr(int_y, x)::decimal FROM statistics_agg_test
-----
-0.9330078226479681  0.9330078226479681  0.9330078226479681  0.9330078226479681
-
-query F
-SELECT corr(DISTINCT y, x)::decimal FROM statistics_agg_test
-----
-0.9326733179802503
-
-query F
-SELECT CAST(corr(DISTINCT y, x) FILTER (WHERE x > 3 AND y < 30) AS decimal) FROM statistics_agg_test
-----
-0.9326733179802503
+INSERT INTO statistics_agg_test (y, x, int_y, int_x, dy, dx) VALUES
+  (1.0,   10.0,    1,   10, 1.0,   10.0),
+  (2.0,   25.0,    2,   25, 2.0,   25.0),
+  (2.0,   25.0,    2,   25, 2.0,   25.0),
+  (3.0,   40.0,    3,   40, 3.0,   40.0),
+  (3.0,   40.0,    3,   40, 3.0,   40.0),
+  (3.0,   40.0,    3,   40, 3.0,   40.0),
+  (4.0,  100.0,    4,  100, 4.0,  100.0),
+  (4.0,  100.0,    4,  100, 4.0,  100.0),
+  (4.0,  100.0,    4,  100, 4.0,  100.0),
+  (4.0,  100.0,    4,  100, 4.0,  100.0),
+  (NULL,  NULL, NULL, NULL, NULL, NULL)
+
+query FFFFFFFFFFFFFFF
+SELECT corr(y, x)::decimal, corr(int_y, int_x)::decimal, corr(y, int_x)::decimal, corr(int_y, x)::decimal, corr(dy, dx),
```

Comment:
I decided to combine corr, covar_pop & covar_samp queries. Please let me know if it is better to keep them separately.